packages/core/src/types/ICamera.ts

```diff
@@ -25,6 +25,8 @@ interface ICamera {
   viewPlaneNormal?: Point3;
   /** Camera viewUp - the direction of viewUP in camera */
   viewUp?: Point3;
+  /** rotation  */
+  rotation?: number;
```

Comment:
It seems you moved the rotation to ViewReference in places, but here it is in ICamera.  I think ICamera makes the most sense, but it would be nice to clarify that.